### ISSUE TITLE
Overfitting 

### ISSUE THREAD

**Issue by kaustubhkmr** [deepconvolution/LipNet#1]
Model is overfitting. 
epoch,acc,loss,val_acc,val_loss

0,0.880010190217,0.408623725504,0.255062849162,6.83833348285

1,0.990076669255,0.0300159885314,0.244413407821,8.60203889262

2,0.994348501019,0.0168089184037,0.359811452514,5.12557166382

3,0.99496554736,0.0153266232897,0.34532122905,6.72021339459

4,0.99537800854,0.0151334269571,0.334846368715,5.99330695598

5,0.995949354807,0.0122128902211,0.354923184358,6.25637885545
